### ISSUE TITLE
[FEATURE] Force transcode option in DIVE Desktop

### ISSUE THREAD

**Issue by subdavis** [Kitware/dive#922]
**Is your feature request related to a problem? If so, Please describe.**
Easiest option for correcting misbehaving videos.

**Comment by BryonLewis** [comment]
Curious if this should be done at import time, or if we should provide a way to do it afterwards, or both.  Most of the time they aren't going to know it needs to be transcoded until after they launch it.  Should we have them delete and re-import or enable a way to run it on a currently existing dataset?

**Comment by subdavis** [comment]
IMO, probably both, but at import time to start with.  We can't put any more  buttons into the toolbar until we figure out how to fix that, and I don't really want to put that on the resents list.

**Comment by BryonLewis** [comment]
Defining implementation:

- When importing a video on desktop there will now be provided under the advance options similar to the image-sequence glob pattern a switch to force transcoding.
- This switch will only appear when dealing with video files.
- This switch when toggled will add to the existing mediaConvertList which is empty, the current video file.
- If the video was already determined to be web-unsafe the switch will appear but disabled the alert at the top of the screen should be enough to let the user know that it will auto transcode.


Unknowns or decisions that need to be made:
- Stereo?  Do I enable the ability to force transcoding of stereo inputs or is that overkill right now?  Leaning towards not doing it for stereo inputs.

**Comment by subdavis** [comment]
Yeah, don't do it for stereo.  Those probably won't have audio anyway right, if it's coming off a synchronized camera rig?

**Comment by BryonLewis** [comment]
> Yeah, don't do it for stereo. Those probably won't have audio anyway right, if it's coming off a synchronized camera rig?

yeah, although the issue with the NIST didn't include audio either, I think it was just a streaming bad video.
